recipes/ckanweb-deploy-exts.rb

```diff
@@ -129,7 +129,7 @@
 harvest_present = false
 csrf_present = false
 
-node['datashades']['ckan_web']['plugin_app_names'].each do |plugin|
+node['datashades']['ckan_web']['plugin_app_names'].sort.each do |plugin|
```

Comment:
Doesn't ordering of plugins matter?

Reply:
This won't affect the order they get inserted into the config file. That is managed separately elsewhere.

It's just to ensure that ckanext-archiver is installed before ckanext-qa. It's not a good solution, I know.